.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/ambv/black
+    rev: 21.12b0
+    hooks:
+      - id: black
+        language_version: python3.8
+        exclude: ingestion/src/metadata/generated
+  - repo: https://github.com/timothycrosley/isort
+    rev: 5.9.3
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
+        exclude: ingestion/src/metadata/generated
+  - repo: local
+    hooks:
+      - id: google-style-java
+        name: Google Java Code Style for Java
+        description: Formats code in Google's Java codestyle.
+        entry: scripts/format-code.sh
```

Comment:
Is git add for reformatted files, taken care of automatically?

Reply:
Nope, you need to do ```git add``` manually.